os.bpmsuite.common/added/launch/bpmsuite-security-ldap.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+########## Environment Variables ##########
+
+function unset_kie_security_ldap_env() {
+    # please keep these in alphabetical order
+    unset KIE_AUTH_LDAP_ALLOW_EMPTY_PASSWORDS
```

Comment:
those new envs also needs to be added to the image descriptors.

Reply:
True